ssz/sedes/serializable.py

```diff
@@ -214,127 +253,103 @@ def _get_class_namespace(cls):
         yield from cls.__slots__
 
 
-class SerializableBase(abc.ABCMeta):
+class MetaSerializable(abc.ABCMeta):
 
-    def __new__(cls, name, bases, attrs):
-        super_new = super(SerializableBase, cls).__new__
+    def __new__(mcls, name, bases, namespace):
+        declares_fields = "fields" in namespace
 
-        serializable_bases = tuple(b for b in bases if isinstance(b, SerializableBase))
-        has_multiple_serializable_parents = len(serializable_bases) > 1
-        is_serializable_subclass = any(serializable_bases)
-        declares_fields = 'fields' in attrs
+        if declares_fields:
+            has_fields = True
```

Comment:
What if `len(namespace['fields'] == 0`.  Should that be taken into account here?

Reply:
In this case the container initialization two lines below would fail and a `TypeError` would be raised. I've added a comment to make it more clear.